Cake.Recipe/Content/toolsettings.cake

```diff
@@ -1,6 +1,9 @@
 public static class ToolSettings
 {
     public static string[] DupFinderExcludePattern { get; private set; }
+    public static string[] DupFinderExcludeFilesByStartingCommentSubstring { get; private set; }
+    public static int? DupFinderDiscardCost { get; private set; }
+    public static bool? DupFinderThrowExceptionOnFindingDuplicates { get; set; }
```

Comment:
Any reason that this isn't a `private set` in the same way as the other properties?